backend/src/modules/boards/services/update.board.service.spec.ts

```diff
@@ -118,7 +119,9 @@ describe('UpdateBoardService', () => {
 		it('should call slackSendMessageService.execute once with slackMessageDto', async () => {
 			// Create a fake board object with the specified properties
 			const board = BoardFactory.create();
-			board.team = { name: 'xgeeks' };
+			const team = TeamFactory.create({ name: 'xgeeks' });
+
+			board.team = team;
```

Comment:
```suggestion
			board.team = TeamFactory.create({ name: 'xgeeks' });
```

I think you don't need to create a new variable